packages/playground/src/components/contracts_list/contracts_table.vue

```diff
@@ -155,6 +190,23 @@ const props = defineProps({
   },
 });
 
+const rentContracts = ref<NormalizedContract[]>([]);
+const nodeIdsInRentContracts = ref<number[]>([]);
+
+onMounted(() => {
+  rentContracts.value = props.contracts.value.filter(contract => contract.type === ContractType.RENT);
```

Comment:
We are already passing the contracts based on their types, namely [name, rent, node]. Instead of individually creating tables for each contract type using a `for loop`, you can directly render them using a single loop within the template. This eliminates the need for multiple instances of the `contract-table` component.

For example:

```vue
<template>
  <div>
    <contract-table v-for="(contractType, index) in contracts" :key="index" :contracts="contractType" />
  </div>
</template>

<script>
import { ref, onMounted } from 'vue';

export default {
  setup() {
    const contracts = ref([]);

    onMounted(() => {
      contracts.value = getContracts(); // Returns an array of contracts grouped by type [NameContract[], NodeContract[], RentContract[]]
    });

    return {
      contracts,
    };
  },
};
</script>